src/main/features/dashboard/views/macro/claimStatus/claimantResponse.njk

```diff
@@ -0,0 +1,30 @@
+{% macro claimantRejectedDefendantAsCompanyOrOrganisationResponse() %}
+<p>
+  {{ t('You need to send the defendant’s financial details to the court.') }}
+</p>
+{% endmacro %}
+
+{% macro claimantRejectedDefendantAsCompanyOrOrganisationResponseDetailedStatus(claim) %}
+<h2 class="heading-small">You need to send the defendant's financial details to the court</h2>
+
+<p>Make sure you include your claim number: {{claim.claimNumber}}.</p>
+<p>The court will review the details to see if the defendant can afford your repayment plan.</p>
+
+<h3 class="heading-small">Use this address</h3>
+<p>The defendant should have sent you the details - send them to:</p>
+<p>
+  Civil Money Claims<br/>
```

Comment:
Please don't use br tags. please use the pre tags. An example can be found in support.njk